app/_includes/md/konnect/generic-sso.md

```diff
@@ -0,0 +1,122 @@
+<!-- used in the Dev Portal generic SSO how to and the Org generic SSO how to -->
+
+## Prerequisites
+
+* {{site.konnect_short_name}} must be added to your IdP as an application
+* Users that need to use SSO are added to your IdP tenant
+* Claims are set up in your IdP
+
+
+## Set up SSO in {{site.konnect_short_name}}
+{% navtabs %}
+{% navtab OIDC%}
+{% if include.desc == "Konnect Org" %}
+1. In [{{site.konnect_saas}}](https://cloud.konghq.com/login), click {% konnect_icon organizations %} **Organization** > **Settings**, and then click the **Authentication Scheme** tab.
+{% endif %}
+{% if include.desc == "Dev Portal" %}
```

Comment:
Same comment here as the previous, but everything in this `if` tag is only pulled into the Dev Portal SSO docs, not the Konnect org SSO docs.

Reply:
Very powerful! Thank you!